Sandru_Dumitrel-Daniel/Task_Viewer/src/components/task-viewer/TaskViewer.js

```diff
@@ -0,0 +1,56 @@
+import "./TaskViewer.css";
+import TaskCard from "../task-card/TaskCard";
+
+function TaskViewer(props) {
+  return (
+    <div className="task-viewer">
+      <h1 className="header">Task Manager</h1>
+      <div className="tasks-container">
+        <TaskCard
```

Comment:
Dupa ce se schimba `data` din `App.js` in array se parseaza si se returneaza cate un `TaskCard `pentru fiecare obiect.
```javascript
{
   data?.map((taskCardData) => {
       return (
           <TaskCard
                id={taskCardData.id}
                status={taskCardData.status}
                name={taskCardData.name}
                dueDate={taskCardData.dueDate}
             />
           );
   })
}
```